frontend/src/components/common/header/header.tsx

```diff
@@ -20,15 +24,22 @@ const Header: FC = () => {
           <Image width="150" height="94" src={logo} alt="logo" />
         </div>
         <div className={styles.userWrapper}>
-          <button onClick={handlePopupOper} className={styles.button}>
-            <Image
-              width="50"
-              height="50"
-              src={defaultUserAvatar}
-              alt="user avatar"
-              isCircular
-            />
-          </button>
+          {hasUser ? (
+            <button onClick={handlePopupOper} className={styles.button}>
```

Comment:
```suggestion
            <button onClick={handlePopupOpen} className={styles.button}>
```
could you please fix callback name

Reply:
done